src/Courses/TopicSettings/ProblemSettings.tsx

```diff
@@ -304,9 +185,11 @@ export const ProblemSettings: React.FC<ProblemSettingsProps> = ({selected, setSe
                             </Grid>
                         )}
                     </Grid><Grid item xs={12}>
-                        {
-                            rendererPreview()
-                        }
+                        <RendererPreview 
+                            defaultPath={topic.topicTypeId === TopicTypeId.EXAM ? 
+                                additionalProblemPaths?.[0].path || '' : 
```

Comment:
don't we have something better to fall back to besides empty-string? Will there be a value for webworkQuestionPath if the topicType is EXAM?

Reply:
If the array is empty this would result in a null pointer exceptions?